admin/settings.php

```diff
@@ -126,23 +126,30 @@ public function init_settings() {
 				if(!isset($field["register"]) || $field["register"]) {
 					register_setting($this->option_group, $this->prefix.$field["name"]);
 				}
-				
+
 			}
 		}
 	}
-	
+
 	/**
 	 * Add Settings submenu and page
 	 *
 	 * @since  0.10
 	 * @return void
 	 */
 	public function add_settings_menu() {
+
+		$cap = RUA_App::CAPABILITY;
+		if ( ! is_multisite() && is_super_admin() && ! current_user_can( $cap ) ) {
+			// Replicate multisite super admin behavior for single installations.
+			$cap = 'manage_options';
```

Comment:
No problemo, please check the commits and take from them what you want :).

For the visibility of this capability I'd suggest to add a filter for this.
`members_get_capabilities` is one that is widely used among a lot of plugins. This way it will be shown at a lot of plugins.
Also, it's used for the post type so it will show up anywhere if these are added to the list.
Currently RUA also doesn't show Gravity Forms capabilities for example, the `members_get_capabilities` can fix this.
Want me to look into this?

On networks non-super admins won't have access by default. IMO this is preferable since a site admin in a network situation doesn't have user access by default.
If we want to validate against `delete_users` in networks this can be fixed easily by adding this check in the filter.

Let me know!